exchanges/order/orders.go

```diff
@@ -709,6 +709,10 @@ func (t Type) String() string {
 		return "MMP"
 	case MarketMakerProtectionAndPostOnly:
 		return "MMP_AND_POST_ONLY"
+	case TWAP:
+		return "TWAP"
+	case Chase:
+		return "CHASE"
```

Comment:
While you are here can you also expand coverage for case TakeProfit, TakeProfitMarket, Liquidation please